tests/app/config/config.yml

```diff
@@ -5,3 +5,7 @@ sulu_core:
                  internal:
                      path: "%kernel.root_dir%/../../src/Sulu/Bundle/CoreBundle/Content/templates"
                      type: "page"
+
+services:
+    sulu_document_manager_test.property_encoder:
```

Comment:
@chirimoya That's how we have to redefine private services now, in order to be able to test them using `getContainer()->get()`. The question is `sulu_document_manager_test` prefix ok for this.